ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py

```diff
@@ -2264,16 +2264,16 @@ def test_paddle_conj(
     )
 
 
-# floor_
+# log2
```

Comment:
Seems like you're entirely removing `floor_` tests

Reply:
Sorry I wasn't renaming it I guess when I created the PR floor_ was not implemented in test. And my IDE automatically sync with latest repo.I wasn't aware of it I removed it and thought I did something. But now I have added floor_ you can review again and let me know.